lib/ansible/plugins/cliconf/ios.py

```diff
@@ -64,11 +64,22 @@ def get_config(self, source='running'):
 
     @enable_mode
     def edit_config(self, command):
-        for cmd in chain(['configure terminal'], to_list(command), ['end']):
-            self.send_command(to_bytes(cmd))
+        for cmd in chain([b'configure terminal'], to_list(command), [b'end']):
```

Comment:
This shouldn't be byte strings (they cast to bytes later anyway), and byte strings will break Python 3.5